scripts/lint-pytest

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+RC=0
+
+pip install -q -r tests/requirements.txt || exit 1
```

Comment:
I would not do the pip install here or in the script below.  This is mainly to make it quick to run these scripts locally before committing and not have it run through a pip install every time.  CircleCI and the dev image should install these things separately in a separate step and be cached.

Reply:
I added this in these scripts because we're using different pylint versions for the python files and the pytest files for python v.2.7 and v3.6 compatibility. So this just explicitly ensures that the correct version is installed when running pylint locally in the dev image.